### DISCUSSION TITLE
Inconsistent device-list

### DISCUSSION THREAD

**Discussion by kartun83** [zepp-health/zeppos-forum#213] [Developer Feedback]
Hello.
Page at :
https://docs.zepp.com/docs/reference/related-resources/device-list/

Missing:
{
                    "name": "Amazfit GTR 4 Limited Edition",
                    "deviceSource": 7864576
                },
                {
                    "name": "Amazfit GTR 4 Limited Edition",
                    "deviceSource": 7864577
                }

**Comment by MyoungXUE** [comment]
Thanks for the heads up, we'll be fixing this soon!

**Comment by MyoungXUE** [comment]
Hi, thanks again for the feedback, the docs have been updated!

https://docs.zepp.com/docs/reference/related-resources/device-list/
